pkgs/racket-doc/scribblings/raco/test.scrbl

```diff
@@ -407,4 +407,15 @@ with @exec{raco test} should also use this library to log test results.
  counted in the test log, such as when testing a custom check's failure
  behavior.}
 
+@defparam[current-test-invocation-directory test-invocation-directory (or/c #f path-string?) #:value #f]{
```

Comment:
And this should be documented like `current-directory` (notice how the setter and getter have different contracts), if the above change is made.